src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(fromFileName, toFileName);
+        writeReportToFile(report, toFileName);
+    }
+
+    private String createReport(String fromFileName, String toFileName) {
```

Comment:
```suggestion
    private String createReport(String[] reportData) {
```
move getting data from file to another method
refactor this method to just create report from data for report